Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -21,7 +21,7 @@ For installing ``kubeadm`` and for more provisioning options please refer to
 
    Cilium's kube-proxy replacement depends on the :ref:`host-services` feature,
    therefore a v4.19.57, v5.1.16, v5.2.0 or more recent Linux kernel is required.
-   We recommend a v5.3 or even more recent Linux kernel such as v5.8 as Cilium
+   We recommend a v5.3 or even more recent Linux kernel such as v5.7 as Cilium
```

Comment:
Why changing? 5.8 will have the getpeername hook which is why I stated it here.

Reply:
Considering that it will take a while until v5.8 has been released, it might discourage users to try the replacement. We could update the docs after the release.

The same logic applies to 5.7 - it hasn't been released yet. However, considering that it's in rc7, it should be released before we release Cilium v1.8.

Reply:
This would probably be less ambiguous if we just stated the facts:

"Linux kernels 5.3 and 5.8 add additional features that Cilium can use to further optimize the kube-proxy replacement implementation."

Reply:
Agreed, will add it in a separate PR.